qip-0009.md

```diff
@@ -0,0 +1,529 @@
+ ```
+  QIP: 9
+  Layer: Consensus (hard fork)
+  Title: Interlinks in Quai
+  Author: gameofpointers <gameofpointers+qip@quai.org>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/quainetwork/qips/wiki/Comments:QIP-0009
+  Status: Draft
+  Type: Standards Track
+  Created: 2023-11-16
+  License: BSD-2-Clause
+ ```
+
+## Abstract
+
+This QIP defines a block interlink structure to be used in Quai as described in
+[1] with few modifications.
+
+## Motivation
+
+To Prove/Verify the Proof Of Work done on Quai since the genesis using a
```

Comment:
I have a nit-picky, but important comment. This QIP is not proving or verifying anything.

This QIP defines a protocol extension to make it possible for nodes to provide concise proofs (NiPoPoWs) of the work/entropy since any given block.

Reply:
Agreed

Reply:
This is just a motivation section

Reply:
Made changes to this to make it smaller and  cleaner